README.md

```diff
@@ -1,4 +1,23 @@
-
 # npm-force-upgrade
+> Force upgrade each package in package.json to the latest version
+
+```sh
+$ npm-force-upgrade
+```
+
+This package provides a script that serves as a temporary fix for `npm update` not writing package versions to `package.json`. Install it using npm:
```

Comment:
I would remove this line. This is not so much a temporary fix, rather, a permanent fix because I doubt NPM will ever change.

Reply:
right, i also tried `ncu` which doesn't seem to overwrite wildcard versions on upgrade, but if either of those issues were ever addressed this package would end up being obsolete.